### ISSUE TITLE
Adjusting Loss Function and Class Imbalance

### ISSUE THREAD

**Issue by arattari** [dingjiansw101/AerialDetection#51]
So, if I wanted to adjust the loss function to account for class imbalance ..... where would I do that? Anyone have any thoughts on compensating for class imbalance in the training data?

**Comment by mmoghadam11** [comment]
> So, if I wanted to adjust the loss function to account for class imbalance ..... where would I do that? Anyone have any thoughts on compensating for class imbalance in the training data?

did you find out what to do???
